<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FManufacturerEdit</class>
 <widget class="QDialog" name="FManufacturerEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manufacturer edit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../res/hacc.qrc">
    <normaloff>:/items/manufacturer.png</normaloff>:/items/manufacturer.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label3">
       <property name="text">
        <string>Icon</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="buddy">
        <cstring>leName</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="ui::icons::WIconSelect" name="pwIcons" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="leName"/>
       </item>
       <item>
        <widget class="QToolButton" name="tbClearName">
         <property name="toolTip">
          <string>Clear name</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../../res/hacc.qrc">
           <normaloff>:/main/icons/edit-clear-locationbar-rtl.png</normaloff>:/main/icons/edit-clear-locationbar-rtl.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Description</string>
       </property>
       <property name="buddy">
        <cstring>teDescription</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="tbClearDescription">
       <property name="toolTip">
        <string>Clear description</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res/hacc.qrc">
         <normaloff>:/main/icons/edit-clear-locationbar-rtl.png</normaloff>:/main/icons/edit-clear-locationbar-rtl.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Logo</string>
       </property>
       <property name="buddy">
        <cstring>tbLoadLogo</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QToolButton" name="tbClearLogo">
       <property name="toolTip">
        <string>Clear logo</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res/hacc.qrc">
         <normaloff>:/main/icons/edit-clear-locationbar-rtl.png</normaloff>:/main/icons/edit-clear-locationbar-rtl.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QToolButton" name="tbLoadLogo">
       <property name="toolTip">
        <string>Open logo</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res/hacc.qrc">
         <normaloff>:/main/icons/document-open-data.png</normaloff>:/main/icons/document-open-data.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPlainTextEdit" name="teDescription">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="3">
      <widget class="ui::images::WImage" name="lLogo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>No image</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ui::icons::WIconSelect</class>
   <extends>QWidget</extends>
   <header>wiconselect.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ui::images::WImage</class>
   <extends>QLabel</extends>
   <header>wimage.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leName</tabstop>
  <tabstop>pwIcons</tabstop>
  <tabstop>teDescription</tabstop>
  <tabstop>tbLoadLogo</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>tbClearName</tabstop>
  <tabstop>tbClearDescription</tabstop>
  <tabstop>tbClearLogo</tabstop>
 </tabstops>
 <resources>
  <include location="../../../res/hacc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>tbClearName</sender>
   <signal>clicked()</signal>
   <receiver>leName</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbClearDescription</sender>
   <signal>clicked()</signal>
   <receiver>teDescription</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FManufacturerEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FManufacturerEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
